### ISSUE TITLE
[Feature Request] More flexible audio playback speed

### ISSUE THREAD

**Issue by gitoss** [hardcore-sushi/DroidFS#247]
I understand the built-in apps like the picture viewer are probably meant to be basic. 

However, the "Speed" option for audio files seems more like a proof of concept as there are only 3 options available: 0.25x - 0.5x - 0.75x.

Ymmv, but when listening to audio even 0.75x generates too much distortion, the settings I use in other apps for voice playback are mostly 0.80x, 0x85x, 0.90x. 

Feature request: Add more fixed speed settings to select like the ones above, or add a free-form speed setting - or maybe a dial/slider in 0.05x steps.

Thanks!

**Comment by hardcore-sushi** [comment]
Yes that would be nice. I think it has already been requested for the video player.
